src/Api/DataProvider/CoverCollectionDataProvider.php

```diff
@@ -61,7 +62,7 @@ public function supports(string $resourceClass, string $operationName = null, ar
      * {@inheritdoc}
      *
      * @throws RequiredParameterMissingException
-     * @throws UnknownIdentifierTypeException
+     * @throws UnknownIdentifierTypeException|IdentifierCountExceededException|UnknownImageSizeException
```

Comment:
Normally we have one line for each exception?

> It is also RECOMMENDED that this tag occurs for every occurrance of an exception

Reply:
Changed